client/view/base/index.css

```diff
@@ -1,7 +1,83 @@
-@import "./reset.css";
+@import "./BrowsersStat.css";
+@import "./Article.css";
+@import "./Form.css";
+@import "./Introduction.css";
+@import "./Badge.css";
+@import "./QueryLinks.css";
 
 @media (prefers-reduced-motion: reduce) {
   * {
     transition: none !important;
   }
 }
+
+body {
+  display: flex;
+  height: 100vh;
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Fira Sans",
+    "Droid Sans",
+    "Helvetica Neue",
+    sans-serif;
+  color: var(--text-primary);
+  background-color: var(--main-bg);
+  accent-color: var(--accent);
+}
+
+p {
+  font-size: 16px;
+  line-height: 22px;
+  letter-spacing: 0.01em;
+}
+
+.Link {
+  padding-bottom: 0.2em;
+  margin-bottom: -0.2em;
+  color: var(--text-primary);
+  text-decoration: none;
+  background-image: linear-gradient(to top, var(--text-primary) 1px, transparent 1px, transparent);
+  background-position: 0 1.3em;
+  transition: background-position var(--time);
+
+  --time: 0.3s;
+}
+
+@media (hover) {
+  .Link:hover {
+    background-position: 0 1.1em;
+  }
+}
+
+code {
+  /* stylelint-disable-next-line font-family-no-duplicate-names */
+  font-family: monospace, monospace;
```

Comment:
Why do we need 2 fonts and disable rule?

Reply:
Because with one `monospace` browser make the `font-size` smaller. See my [example](https://codepen.io/ariarzer/pen/qBopLLp).

Reply:
Crazy hack and crazy browser behaviour

https://github.com/necolas/normalize.css/issues/519#issuecomment-197131966

Let’s disable rule in Stylelint config

Reply:
No, it need just in this case. Move it to the config will be bad. It is a local hack.

Reply:
Let's add a comment with a link to hack below rule disabling to explain why we did it